src/Components/Table/Table.tsx

```diff
@@ -137,9 +137,14 @@ const TableData = (props: StudentProfilePropsType) => {
     ['Created By', true],
     ['Game Name', true],
     ['Time', true],
-    ['Date', true]
+    ['Date', true],
```

Comment:
why we have array over here can't we use object like this for columns
const columns = [
{label: "Name", key: "name", isSortable: true}
]

you are having 2 iteration over column from prop and sorted column instead we can have just one and use that simply and then you have a consition based on url instead just pass that as props columns only in required pages that's what does this means